Mathlib/SetTheory/Ordinal/NaturalOps.lean

```diff
@@ -86,7 +87,14 @@ instance : WellFoundedLT NatOrdinal :=
   Ordinal.wellFoundedLT
 
 instance : IsWellOrder NatOrdinal (· < ·) :=
-  Ordinal.isWellOrder
+  { }
```

Comment:
What's going on here?

Reply:
An `IsWellOrder α (· < ·)` instance can always be inferred from `WellFoundedLT α` + `LinearOrder α`. Unfortunately, it doesn't seem like I can remove this instance altogether.

I can revert this golf if there's any issues with it.